### ISSUE TITLE
TypeError: 'NoneType' object is not callable in SelfQueryRetriever.from_llm

### ISSUE THREAD

**Issue by levalencia** [hwchase17/langchain#7468]
### System Info

langchain 0.0.228

### Who can help?

@dev2049 

### Information

- [ ] The official example notebooks/scripts
- [X] My own modified scripts

### Related Components

- [X] LLMs/Chat Models
- [ ] Embedding Models
- [ ] Prompts / Prompt Templates / Prompt Selectors
- [ ] Output Parsers
- [ ] Document Loaders
- [X] Vector Stores / Retrievers
- [ ] Memory
- [ ] Agents / Agent Executors
- [ ] Tools / Toolkits
- [ ] Chains
- [ ] Callbacks/Tracing
- [ ] Async

### Reproduction

Code is very similar to existing example, instead of ``` Pinecone.from_documents``` I use ```Pinecone.from_documents.from_existingindex```

```
    llm = AzureChatOpenAI(
        openai_api_base=OPENAI_DEPLOYMENT_ENDPOINT,
        openai_api_version=OPENAI_API_VERSION ,
        deployment_name=OPENAI_DEPLOYMENT_NAME,
        openai_api_key=OPENAI_API_KEY,
        openai_api_type = OPENAI_API_TYPE ,
        model_name=OPENAI_MODEL_NAME,
        temperature=0)    
    embed = OpenAIEmbeddings(deployment=OPENAI_EMBEDDING_DEPLOYMENT_NAME, model=OPENAI_EMBEDDING_MODEL_NAME, chunk_size=1)
    user_input = get_text()   
    
    metadata_field_info = [
        AttributeInfo(
            name="IdentityId",
            description="The id of the resident",
            type="string",
        ),
        AttributeInfo(
            name="FirstName",
            description="The first name of the resident",
            type="string",
        ),
        AttributeInfo(
            name="LastName",
            description="The last name of the resident",
            type="string",
        ),
        AttributeInfo(        
            name="Gender", 
            description="The gender of the resident", 
            type="string"
        ),
        AttributeInfo(        
            name="Birthdate", 
            description="The birthdate of the resident", 
            type="string"
        ),
        AttributeInfo(
            name="Birthplace",
            description="The birthplace of the resident",
            type="string"
        ),
        AttributeInfo(
            name="Hometown",
            description="The hometown of the resident",
            type="string"
        )
    ]
    document_content_description = "General information about the resident for example: Phone number, Cell phone number, address, birth date, owned technologies, more about me, edication, college name, past occupations, past interests, whether is veteran or not, name of spourse, religious preferences, spoken languages, active live description, retired live description, accomplishments, marital status, anniversay date, his/her typical day, talents and hobbies, interest categories, other interest categories, favorite actor, favorite actress, etc"
    llm = OpenAI(temperature=0)
    vectordb = Pinecone.from_existing_index("default",embedding=embed, namespace="profiles5")
  
    retriever = SelfQueryRetriever.from_llm(
        llm, vectordb, document_content_description, metadata_field_info, verbose=True
    )

    qa_chain = RetrievalQA.from_chain_type(llm,retriever=retriever)
    response = qa_chain.run(user_input)
    st.write(response)

```

Error: 

TypeError: 'NoneType' object is not callable
Traceback:
File "C:\Users\xx\anaconda3\envs\cnChatbotv3\Lib\site-packages\streamlit\runtime\scriptrunner\script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
File "C:\Users\xx\repos\cnChatbotv1\app\pages\07Chat With Pinecone self-querying.py", line 151, in <module>
    main()
File "C:\Users\xx\repos\cnChatbotv1\app\pages\07Chat With Pinecone self-querying.py", line 142, in main
    retriever = SelfQueryRetriever.from_llm(
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "C:\Users\xx\anaconda3\envs\cnChatbotv3\Lib\site-packages\langchain\retrievers\self_query\base.py", line 149, in from_llm
    llm_chain = load_query_constructor_chain(
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "C:\Users\xx\anaconda3\envs\cnChatbotv3\Lib\site-packages\langchain\chains\query_constructor\base.py", line 142, in load_query_constructor_chain
    prompt = _get_prompt(
             ^^^^^^^^^^^^
File "C:\Users\xx\anaconda3\envs\cnChatbotv3\Lib\site-packages\langchain\chains\query_constructor\base.py", line 103, in _get_prompt
    output_parser = StructuredQueryOutputParser.from_components(
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "C:\Users\xx\anaconda3\envs\cnChatbotv3\Lib\site-packages\langchain\chains\query_constructor\base.py", line 60, in from_components
    ast_parser = get_parser(
                 ^^^^^^^^^^^
File "C:\Users\xx\anaconda3\envs\cnChatbotv3\Lib\site-packages\langchain\chains\query_constructor\parser.py", line 148, in get_parser
    transformer = QueryTransformer(

### Expected behavior

response to the query should be returned.

**Comment by baskaryan** [comment]
@levalencia guessing you need to `pip install lark`. The unintuitive error message should be fixed in 0.0.229 (see #7465)

**Comment by levalencia** [comment]
yeah, the error was due to two llms being insantiated, and after fixing that error I got another error regarding lark which was fixed with installing.

thanks a lot
